sdk/mobilenetwork/Azure.ResourceManager.MobileNetwork/tests/autorest.tests.md

```diff
@@ -0,0 +1,13 @@
+# Generated code configuration
```

Comment:
remove this file and all generated samples under `tests/Generated/Samples/` folder